lib/routes/root.js

```diff
@@ -35,9 +35,8 @@ module.exports = {
     var gitDir = path.resolve(__dirname, '..', '..', '.git');
     exec('git rev-parse HEAD', { cwd: gitDir }, function(err, stdout) {
       commitHash = stdout.replace(/\s+/, '');
-      var configPath = path.join(gitDir, 'config');
       var cmd = 'git config --get remote.origin.url';
-      exec(cmd, { env: { GIT_CONFIG: configPath } }, function(err, stdout) {
+      exec(cmd, { cwd: gitDir }, function(err, stdout) {
```

Comment:
The auth server uses the env variables. I feel that was changed specifically because of some security concern...